tests/unit/style/custom_properties.rs

```diff
@@ -9,13 +9,33 @@ use style::context::QuirksMode;
 use style::custom_properties::{
     ComputedCustomProperties, CustomPropertiesBuilder, Name, SpecifiedValue,
 };
+use style::font_metrics::FontMetrics;
 use style::media_queries::{Device, MediaType};
+use style::properties::style_structs::Font;
 use style::properties::{CustomDeclaration, CustomDeclarationValue};
 use style::rule_tree::CascadeLevel;
+use style::servo::media_queries::FontMetricsProvider;
 use style::stylesheets::layer_rule::LayerOrder;
 use style::stylist::Stylist;
+use style::values::computed::CSSPixelLength;
```

Comment:
`Length` is shorter (they are synonyms).

Reply:
Makes sense. I always forget if `Length` is in stylo or layout. Fixed.